requirements.txt

```diff
@@ -3,3 +3,4 @@ ortools==9.4.1874
 pandas==1.5.2
 regex==2022.10.31
 Unidecode==1.3.6
+Werkzeug==2.2.2
```

Comment:
Why did you added this lib?

Reply:
I added this library because I encountered an error in Flask: 'ImportError: cannot import name 'url_quote' from 'werkzeug.urls'. After researching, I discovered that the issue was related to the Werkzeug dependency in Flask. The current version 3.0.0 does not work properly, so I had to manually set the version to an earlier one to get my project functioning correctly.

Reply:
Ummm, in my PC the flask is in 2.2.3 version. Thanks for this